src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -263,7 +263,8 @@ class CreateFastaFromHvcf : CliktCommand( help = "Create a FASTA file from a h.v
      * This is almost identical to how fastas were exported in the original version of the pipeline.
      */
     fun writeHaplotypeSequence(outputFileWriter: BufferedWriter, haplotypeSequences: List<HaplotypeSequence>, exportFullIdLine : Boolean = true) {
-        for(hapSeq in haplotypeSequences) {
+        // Make this a set to remove duplicates
+        for(hapSeq in haplotypeSequences.toSet()) {
```

Comment:
Should this be sorted?  Just so the output file is consistent given an input?

Reply:
hapSeq is the data class HaplotypeSequence.  If you sort by ID, you are sorting by values of the sequence hash.  That won't get you the data in the order of chrom/pos, so I'm not sure this is useful.

Reply:
I could pehaps change to a LinkedHashSet or something that does maintain order.  I will test with that.